test/functional/tests/fault_injection/test_standby_activate_the_same_cache_path.py

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+import pytest
+
+from api.cas import casadm
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_utils.size import Size, Unit
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.nand, DiskType.optane]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_standby_activate_the_same_cache_path():
```

Comment:
Add test description and pass criteria.
It's most offten in that form. Check also other tests for that.
    
```
    """
    title: <title>
    description: |
        <desc>
    pass_criteria:
      - <crit>
      - <crit>
    """
```

Reply:
done